tailwind.config.ts

```diff
@@ -99,6 +99,7 @@ const config: Config = {
         },
       },
       screens: {
+        sm500: '31.5rem',
```

Comment:
```suggestion
        "sm-500": '31.5rem',
```
Может разделить, чтобы симпатичнее смотрелось, что думаешь?